client/signup/steps/domains/index.jsx

```diff
@@ -602,12 +602,26 @@ class DomainsStep extends React.Component {
 	};
 
 	getSubHeaderText() {
-		const { flowName, isAllDomains, siteType, translate } = this.props;
+		const {
+			flowName,
+			isAllDomains,
+			siteType,
+			stepSectionName,
+			isReskinned,
+			translate,
+		} = this.props;
 
 		if ( isAllDomains ) {
 			return translate( 'Find the domain that defines you' );
 		}
 
+		if ( isReskinned ) {
+			return (
+				! stepSectionName &&
+				translate( "Enter your site's name or some descriptive keywords to get started" )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Enter your site's name or some keywords that describe it to get started.' )` *ES Score: 6*